Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
rubocop wants a magic comment concerning frozen strings. I opt for false, in part because I'm uncertain of all the ramifications and since this is homework, I don't want to deal with a sidetrack into frozen string literals.
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment